Example.ipynb

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    random.seed(0)
</pre><p>Nitpick: change seed to some different number, please!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDolinski/jupyter-deployment/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>